actionpack/lib/action_dispatch/middleware/flash.rb

```diff
@@ -241,10 +234,19 @@ def keep(k = nil)
         k ? self[k] : self
       end
 
+      # Immediately deletes the single flash entry. Use this method when you want remove the message within the current action. See also <tt>discard</tt>.
```

Comment:
For API, we usually try to wrap the lines around 80-100 characters.

Also, if you wrap something with fixed-width (`<tt></tt>` or `+foo+`), RDoc won't be able to create a link to the reference. This is an old, hard-to-fix bug 😢